framework/components/ecl-dialogs/ecl-dialogs.scss

```diff
@@ -35,6 +39,23 @@
   z-index: map-get($ecl-z-index, 'highlight');
 }
 
+.ecl-dialog__overlay--blue {
+  background-color: rgba($ecl-color-primary, 0.95);
+}
+
+.ecl-dialog--wide .ecl-dialog__body {
+  overflow: initial;
+  width: 100%;
+}
+
+.ecl-dialog__title {
+  clear: both;
+  color: #fff;
+  font-size: map-get($ecl-font-size, 'l');
+  margin: 0;
+  padding: 2em 0;
```

Comment:
Please replace `2em` by the nearest value available in `$ecl-spacing` (-> probably `map-get($ecl-spacing, 'l')`)